bin/p2-launch/main.go

```diff
@@ -56,7 +56,7 @@ func main() {
 	}
 
 	podFactory := pods.NewFactory(*podRoot, types.NodeName(*nodeName))
-	pod := podFactory.NewPod(manifest.ID())
+	pod := podFactory.NewPod(manifest.ID(), nil)
```

Comment:
in the interest of keeping the PR small some tools are changed to just have the old behavior. Later on there may be a need to `p2-launch` a pod with a UUID, although there are questions that are raised there (like `p2-launch` needing to talk to consul)